zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/instance/ElementInstance.java

```diff
@@ -235,11 +234,15 @@ public void setUserTaskKey(final long userTaskKey) {
     userTaskKeyProp.setValue(userTaskKey);
   }
 
-  public String getExecutionListenerType() {
-    return BufferUtil.bufferAsString(executionListenerTypeProp.getValue());
+  public long getExecutionListenerIndex() {
```

Comment:
:wrench: I guess an integer property should return an `int` value.

```suggestion
  public int getExecutionListenerIndex() {
```